x-pack/plugins/fleet/server/services/telemetry/fleet_usage_sender.ts

```diff
@@ -126,6 +128,13 @@ export class FleetUsageSender {
       upgradeDetails.forEach((upgradeDetailsObj) => {
         core.analytics.reportEvent(FLEET_AGENTS_EVENT_TYPE, { upgrade_details: upgradeDetailsObj });
       });
+
+      appContextService
+        .getLogger()
+        .debug(() => 'Integrations details telemetry: ' + JSON.stringify(integrationsDetails));
+      core.analytics.reportEvent(FLEET_INTEGRATIONS_EVENT_TYPE, {
+        integrations_details: integrationsDetails,
```

Comment:
we should report separate events for each `SharedIntegration` like it's done above with `upgradeDetails`